### PR TITLE
PNLS REPORT first indicator innitial commit (WIP)

### PR DESCRIPTION
**Pull request by mozzy11**
Innitial Commit 
PNLS REPORT :  https://docs.google.com/spreadsheets/d/1PJERx8NtiiBCcgk92li46t6SU09yIKH1wejXgte1ugY/edit#gid=1709148033

c @moshonk

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/isanteplusreports/RegisterReports.java [hunk 1]
```diff
@@ -23,13 +24,14 @@ public void registerReports() {
 			String version = ModuleFactory.getModuleById("isanteplusreports").getVersion();
 			String oldversion = Context.getAdministrationService().getGlobalProperty("reports.moduleVersion");
 
-			if (!version.equals(oldversion)) {
+			//if (!version.equals(oldversion)) {
```
Comment (moshonk) [inline api/src/main/java/org/openmrs/module/isanteplusreports/RegisterReports.java:27]: Why would you want to change this? I think it was put in place by design. 
It is OK to comment it during development, but it shouldn't be part of the PR

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/isanteplusreports/RegisterReports.java:27]: Yes @moshonk , ideally i comment it just for development . In the final PR i revert it

Comment (ckemar) [reply api/src/main/java/org/openmrs/module/isanteplusreports/RegisterReports.java:27]: Hello Mose, @moshonk , did that PR take into account the indicator desagragation as well

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/isanteplusreports/RegisterReports.java:27]: oh ,it actually doesnt (i havent taken it into account) , @moshonk   has just given me that comment in slack. 
I will re-work on it

**Comment by mozzy11** [comment]
Hi @ckemar @moshonk  , Am closing this PR. 
Please refereto  https://github.com/IsantePlus/openmrs-module-isanteplusreports/pull/69
